var/spack/repos/builtin/packages/pythia8/package.py

```diff
@@ -83,6 +84,7 @@ class Pythia8(AutotoolsPackage):
     variant("mpich", default=False, description="Multi-threading support via MPICH")
     variant("hdf5", default=False, description="Support the use of HDF5 format")
 
+    depends_on("gzip", when="+gzip")
```

Comment:
Unless this actually calls the gzip command instead of linking libz, we should probably use:
```suggestion
    depends_on("zlib-api", when="+gzip")
```

Reply:
Looks like it's linking libz